pkg/runtime/runtime.go

```diff
@@ -43,6 +43,8 @@ type Metadata struct {
 	//KubeVersion is a SemVer constraint specifying the version of Kubernetes required.
 	KubeVersion string `json:"kubeVersion"`
 	NydusFlag   bool   `json:"NydusFlag"`
+	//ClusterRuntime is a flag to distinguish the runtime for k0s、k8s、k3s
+	ClusterRuntime string `json:"ClusterRuntime"`
```

Comment:
do we need to make those enum as const ?

Reply:
Yep! In our plan, it just for `k0s`、`k3s`、`k8s`, I will modify it to enum!